cannelloni.cpp

```diff
@@ -124,9 +128,10 @@ int main(int argc, char** argv) {
     }
   }
   if (!remoteIPSupplied) {
-    std::cout << "Error: Remote IP not supplied" << std::endl;
-    printUsage();
-    return -1;
+    std::cout << "Remote IP not supplied , will be bound to the first incoming connection" << std::endl;
+  }
+  if (!remoteIPSupplied && clientTimeout == 0) {
+    std::cout << "Remote IP not selected an inactivity connection either. Timeout will be set to default" << std::endl;
```

Comment:
`linfo << "No inactivity timeout for connections specified, using default." << std::endl;`

Otherwise we have almost the same message twice